Makefile

```diff
@@ -41,5 +41,4 @@ check-style:
 test: clean check-syntax-errors check-style
 	@# Launch tests from openfisca_france/tests directory (and not .) because TaxBenefitSystem must be initialized
 	@# before parsing source files containing formulas.
-	pytest
```

Comment:
`openfisca test` only runs the YAML tests, not the python tests, so we should probably keep that